lib/Doctrine/DBAL/Schema/Table.php

```diff
@@ -253,12 +253,12 @@ public function columnsAreIndexed(array $columnsNames)
     }
 
     /**
-     * @param mixed[][] $columnNames
-     * @param string    $indexName
-     * @param bool      $isUnique
-     * @param bool      $isPrimary
-     * @param string[]  $flags
-     * @param mixed[]   $options
+     * @param string[] $columnNames
```

Comment:
> […] except for one case here: https://github.com/doctrine/dbal/blob/4b4d44f90ae6efde2a494b12167348bbc0551e09/lib/Doctrine/DBAL/Schema/Table.php#L274
where it is verified if the variable in the array is type of `string`.

Here, it should be something like `string[]|mixed[][]`. Judging by the code, it allows the following syntax:
```php
_createIndex([
    'column_1',
    'column_2' => [
        'option' => 'value'
    ],
]);
```